libdnf/dnf-reldep.h

```diff
@@ -29,11 +31,16 @@ G_BEGIN_DECLS
 
 G_DECLARE_FINAL_TYPE (DnfReldep, dnf_reldep, DNF, RELDEP, GObject)
 
+DnfReldep   *dnf_reldep_from_pool (Pool              *pool,
+                                   Id                 r_id);
+
 DnfReldep   *dnf_reldep_new       (DnfSack           *sack,
                                    const gchar       *name,
                                    DnfComparisonKind  cmp_type,
                                    const gchar       *evr);
 
 const gchar *dnf_reldep_to_string (DnfReldep         *reldep);
 
+Id           dnf_reldep_get_id    (DnfReldep         *reldep);
```

Comment:
we shouldn't expose libsolv API

Reply:
I'm strictly trying to separate public and private functions. Public API must be complete to be fully useable.
We want to hide libsolv for C developers. But we using libsolv in Python API wrapper.

I see two solution:
- Expose libsolv API.
- Or overwrite the Python API wrapper. Prove it without accessing libsolv API. The same rules for us as for other libdnf C API users.

Reply:
first is not a solution but workaround